docs/localnet.md

```diff
@@ -0,0 +1,55 @@
+# Local Testnets
+
+Prerequisites:
+```bash
+make build docker-build
+```
+
+## Single node deployment
+
+To run IPC in the local rootnet just perform the following :
+```bash
+cargo make --makefile ./infra/Makefile.toml node
+
+```
+
+It will create three docker containers (cometbft, fendermint, and eth-api).
+
+To stop run the following:
+```bash
+cargo make --makefile ./infra/Makefile.toml node-down
+```
+
+## Local 4-nodes deployment
+To run IPC in the local rootnet with 4 nodes perform the following command :
+```bash
+cargo make --makefile ./infra/Makefile.toml testnet
+
+```
+
+To stop the network:
+```bash
+cargo make --makefile ./infra/Makefile.toml testnet-down
+```
+
+The testnet contains four logical nodes. Each node consists of cometbft, fendermint, and ethapi containers.
+The testnet network is `192.167.10.0/24`.
+
+ETH-API is accessible on the following interfaces:
```

Comment:
Do these now work from the host? You can use the IP and the internal port, not the exposed one? That would be pretty cool.

Reply:
Fixed. But we can only use the host interface (127.0.0.1) and the exposed ports. That will be compatible with all operating systems.